include/seastar/core/scheduling.hh

```diff
@@ -181,7 +181,7 @@ inline unsigned long scheduling_group_key_id(scheduling_group_key key) noexcept
 
  * @tparam ConstructorType - the constructor type or in other words the type to be constructed
  * @tparam Tuple - T params tuple type (should be deduced)
- * @tparam size_t...Idx - a sequence of indexes in order to access the typpels members at compile time.
+ * @tparam size_t...Idx - a sequence of indexes in order to access the tuple type members at compile time.
```

Comment:
"Typples" is an interesting mixture, but I'm not sure it shouldn't be just "tuples".

Reply:
Yeah, I really meant to ask about this. I was on the fence.

Reply:
I think the reason I didn't go w/ that is i'd half expect `tuple's members` or `tuple members`.

I'm going to leave this unresolved until someone provides guidance.